lib/modules/manager/npm/update/dependency/index.ts

```diff
@@ -279,7 +279,8 @@ function overrideDepPosition(
       overrideDep = overrideDep[parent]! as Record<string, RecursiveOverride>;
     }
   }
-  const overrideDepName = depName === lastParent ? '.' : depName;
+  const overrideDepName =
+    depName === lastParent ? /* istanbul ignore next */ '.' : depName;
```

Comment:
to hard to test? please add comments when adding new ignores

Reply:
I don't understand how to test thiis one :)